src/main/java/berry/command/FindCommand.java

```diff
@@ -0,0 +1,22 @@
+package berry.command;
+
+import berry.task.TaskList;
+import berry.ui.Ui;
+import berry.storage.Storage;
+import berry.exception.BerryException;
+
+public class FindCommand extends Command {
+    private static String[] keyword;
```

Comment:
Perhaps listOfKeywords may be better?

Reply:
Oh right, thanks for this! I will take extra care to rename my variable names when I change their implementations or type ><